libs/model/src/contest/GetActiveContestManagers.query.ts

```diff
@@ -17,44 +18,43 @@ export function GetActiveContestManagers(): Query<
       const results = await models.sequelize.query<{
         eth_chain_id: number;
         url: string;
+        private_url: string;
         contest_address: string;
         max_contest_id: number;
         actions: Array<z.infer<typeof schemas.ContestAction>>;
       }>(
         `
-  SELECT
-    cn.eth_chain_id,
-    COALESCE(cn.private_url, cn.url) as url,
-    cm.contest_address,
-    co.max_contest_id,
-	  COALESCE(JSON_AGG(ca) FILTER (WHERE ca IS NOT NULL), '[]'::json) as actions
-  FROM "Communities" c
-  JOIN "ChainNodes" cn ON c.chain_node_id = cn.id
-  JOIN "ContestManagers" cm ON cm.community_id = c.id
-  JOIN "ContestTopics" ct ON cm.contest_address = ct.contest_address
-  JOIN (
-      SELECT contest_address,
-      MAX(contest_id) AS max_contest_id,
-      MAX(start_time) as start_time,
-      MAX(end_time) as end_time
-      FROM "Contests"
-      GROUP BY contest_address
-  ) co ON cm.contest_address = co.contest_address
-  LEFT JOIN "ContestActions" ca on (
-    ca.contest_address = cm.contest_address AND
-    ca.created_at > co.start_time AND
-    ca.created_at < co.end_time
-  )
-  WHERE ct.topic_id = :topic_id
-  AND cm.community_id = :community_id
-  AND cm.cancelled = false
-  AND (
-    cm.interval = 0 AND NOW() < co.end_time
-    OR
-    cm.interval > 0
-  )
-  GROUP BY cn.eth_chain_id, cn.private_url, cn.url, cm.contest_address, co.max_contest_id
-      `,
+            SELECT cn.eth_chain_id,
+                   CN.private_url,
+                   CN.url,
+                   cm.contest_address,
```

Comment:
is the different casing going to be an issue?

Reply:
Everything not in quotes gets converted to lowercase anyway so no but for the sake of clarity I will fix this.